src/main/java/Nick.java

```diff
@@ -0,0 +1,142 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.Scanner;
+import java.util.logging.Logger;
+
+public class Nick {
+    public static int taskCount = 0;
+    public static final int TO_OFFSET_IDX = 4;
+    public static final int FROM_OFFSET_IDX = 6;
+    public static final int DEADLINE_OFFSET_IDX = 4;
+    private static final Logger LOGGER = Logger.getLogger(Nick.class.getName());
+
+    public static void main(String[] args) {
+        printIntroName();
+        printIntroMsg();
+
+        boolean hasQuit = false;
+        String command, commandAction;
+        Task[] userTasks = new Task[100];
+        Scanner input = new Scanner(System.in);
+
+        while (input.hasNextLine()) {
+            command = input.nextLine();
+            commandAction = command.split(" ")[0];
+
+            switch (commandAction) {
+            case "bye":
+            case "list":
+            case "mark":
```

Comment:
Perhaps place a "//Fallthrough" comment for case statements without a break statement?